src/radical/pilot/agent/agent_0.py

```diff
@@ -340,77 +353,130 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
-
-        if not os.path.isfile('./services'):
-            return
+        service_descriptions = self._cfg.services
+        services = list()
+        self._service_task_ids = {}
+        for service_desc in service_descriptions:
+            task = dict()
+            task['origin'] = 'services'
+            task['description'] = TaskDescription(service_desc).as_dict()
+            task['state'] = rps.AGENT_STAGING_INPUT_PENDING
+            task['status'] = 'NEW'
+            task['type'] = 'task'
+            task['uid'] = ru.generate_id('services.%(item_counter)04d',
+                                             ru.ID_CUSTOM, ns="TODO_WHATTOUSE")
+            self._service_task_ids.add(task['uid'])
+            task['task_sandbox_path'] = self._sbox
+            task['task_sandbox'] = 'file://localhost/' + self._sbox
+            task['pilot_sandbox'] = self._cfg.pilot_sandbox
+            # task['session_sandbox'] = os.environ['RP_SESSION_SANDBOX']
```

Comment:
@andre-merzky Need help here. How to get these

Reply:
`self._cfg.session_sandbox` etc.  You may want to have a look at an agent config, it contains quite a number of config items you are using throughout the code.

Reply:
ok